include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (440)
+#define OPENTHREAD_API_VERSION (441)
```

Comment:
Please rebase and increment :)

Reply:
Will do. Will wait for other simpler PRs (which also increment `API_VERSION` to be merged) before updating this one.

Reply:
Rebased.